includes/application/handleAppl.php

```diff
@@ -31,6 +31,7 @@
 	$UID = $_SESSION['Username'];
 }
 $USER= str_replace('.','$',$UID );
+$USER= str_replace(' ','_',$USER );
```

Comment:
__[code_sniffer]__
* Expected 0 spaces before closing bracket; 1 found
* No space found after comma in function call

<!-- Comment by [SideCI](https://sideci.com) -->